static/skywire-manager-src/src/app/components/vpn/vpn-helpers.ts

```diff
@@ -0,0 +1,515 @@
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { Router } from '@angular/router';
+import { map, mergeMap } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
+
+import { TabButtonData } from '../layout/top-bar/top-bar.component';
+import { VpnClientService, CheckPkResults } from 'src/app/services/vpn-client.service';
+import { SnackbarService } from 'src/app/services/snackbar.service';
+import GeneralUtils from 'src/app/utils/generalUtils';
+import { VpnServer } from 'src/app/services/vpn-client-discovery.service';
+import { ManualVpnServerData } from './pages/vpn-server-list/add-vpn-server/add-vpn-server.component';
+import { LocalServerData, ServerFlags, VpnSavedDataService } from 'src/app/services/vpn-saved-data.service';
+import { Lists } from './pages/vpn-server-list/vpn-server-list.component';
+import { SelectableOption, SelectOptionComponent } from '../layout/select-option/select-option.component';
+import {
+  EditVpnServerParams,
+  EditVpnServerValueComponent
+} from './pages/vpn-server-list/edit-vpn-server-value/edit-vpn-server-value.component';
+import { EnterVpnServerPasswordComponent } from './pages/vpn-server-list/enter-vpn-server-password/enter-vpn-server-password.component';
+
+/**
+ * Helper functions for the VPN client.
+ */
+export class VpnHelpers {
+  /**
+   * Key for saving in sessionStorage the default tab that should be openned when entering to
+   * the server list.
+   */
+  private static readonly serverListTabStorageKey = 'ServerListTab';
+
+  /**
+   * Pk of the local Skywire visor.
+   */
+  private static currentPk = '';
+  /**
+   * Sets the Pk of the local Skywire visor. Must be called for the vpnTabsData property to
+   * work correctly.
+   */
+  static changeCurrentPk(pk: string): void {
+    this.currentPk = pk;
+  }
+
+  /**
+   * Allows to set the default tab that should be openned when entering to the server list.
+   * It is saved while the tab is openned.
+   */
+  static setDefaultTabForServerList(tab: Lists) {
+    sessionStorage.setItem(VpnHelpers.serverListTabStorageKey, tab);
+  }
+
+  /**
+   * Data for configuring the tab-bar shown in the header of the vpn client pages.
+   */
+  static get vpnTabsData(): TabButtonData[] {
+    const lastServerListTab = sessionStorage.getItem(VpnHelpers.serverListTabStorageKey);
+
+    return [
+      {
+        icon: 'power_settings_new',
+        label: 'vpn.start',
+        linkParts: ['/vpn', this.currentPk, 'status'],
+      },
+      {
+        icon: 'list',
+        label: 'vpn.servers',
+        linkParts: lastServerListTab ? ['/vpn', this.currentPk, 'servers', lastServerListTab, '1'] : ['/vpn', this.currentPk, 'servers'],
+      },
+      {
+        icon: 'settings',
+        label: 'vpn.settings',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
        label: "vpn.settings",
```